spec/models/user_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before { @user = User.new(name: '星宮いちご', email: 'i-hoshimiya@star-light.ac') }
+
+  subject { @user }
+
+  it { should respond_to(:name) }
+  it { should respond_to(:email) }
+
+  it { should be_valid }
+
+  describe 'when name is not present' do
+    before { @user.name = '' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.